crates/fluvio-protocol/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-protocol"
 edition = "2021"
-version = "0.10.3"
+version = "0.11.0"
```

Comment:
the changes doesn't seem like need to bump major version

Reply:
Will do minor!